app/mailers/notifier.rb

```diff
@@ -57,7 +57,7 @@ def order_detail_status_changed(order_detail_id)
     @order_detail = OrderDetail.includes(:order, :order_status, :product).find(order_detail_id)
```

Comment:
Would it be worth it while we're here to switch to using the globalid version where we pass in the order detail itself and let activejob handle the lookup?